skyportal/tests/api/test_candidates.py

```diff
@@ -109,30 +133,72 @@ def test_token_user_post_new_candidate(
         token=upload_data_token,
     )
     assert status == 200
-    assert data["data"]["id"] == candidate_id
 
-    status, data = api("GET", f"candidates/{candidate_id}", token=view_only_token)
+    status, data = api("GET", f"candidates/{obj_id}", token=view_only_token)
     assert status == 200
-    assert data["data"]["id"] == candidate_id
+    assert data["data"]["id"] == obj_id
     npt.assert_almost_equal(data["data"]["ra"], 234.22)
 
-
-def test_cannot_add_candidate_without_filter_id(upload_data_token):
-    candidate_id = str(uuid.uuid4())
     status, data = api(
         "POST",
         "candidates",
         data={
-            "id": candidate_id,
+            "id": obj_id,
             "ra": 234.22,
             "dec": -22.33,
             "redshift": 3,
             "transient": False,
             "ra_dis": 2.3,
+            "filter_ids": [public_filter.id],
             "passed_at": str(datetime.datetime.utcnow()),
```

Comment:
I think filter_ids should be a scalar - filters all run separately and it will be impossible for any two filters to pass the exact same candidate at the same time.

Reply:
Not going to change this here -- do you want me to file an issue?

Reply:
Yes please file an issue

Reply:
Makes sense, but I'd request to leave this as is -- the it's implemented now saves (potentially) a lot of calls on the K side of F if an object passes multiple filters.

Reply:
Okay, will _not_ file an issue, then :wink: